.github/workflows/render-site.yml

```
@@ -12,10 +12,18 @@ on:
     branches: [ main ]
     paths:
       - '**.Rmd'
+      - '**.Qmd'
```

We might want to look into if this is case sensitive. To be safe could just add both here: 
```suggestion
      - '**.Qmd'
      - '**.qmd'
```

---

Same comment about 'Rmd' and 'rmd' of course.